initial-data-load/data/Sample/GeoData.yml

```diff
@@ -1,9 +1,38 @@
 - name: "Slovakia - regions"
   referenceId: "slovakiaRegions"
-  geoJSONFilename: "slovakia_regions.geojson"
+  geoDataFilename: "slovakia_regions.geojson"
+  format: "geojson"
+  featureIds: [{property: "name", values: ["Bratislavský", "Trnavský", "Trenčiansky", "Nitriansky", "Žilinský", "Banskobystrický", "Prešovský", "Košický"]}, { property: "fullname", values: ["Bratislavský kraj", "Trnavský kraj", "Trenčiansky kraj", "Nitriansky kraj", "Žilinský kraj", "Banskobystrický kraj", "Prešovský kraj", "Košický kraj"]}] 
+  attributeIds: [] 
+  geometryDataTypes: "regions"
+  geoMetadata: 
```

Comment:
Should this be called geoMetadata or metadata here? On frontend I will then return geoMetadata for sure, otherwise these will mess with each other.

Reply:
metadata, according to the task

Reply:
I fixed that.